OIPA/api/v3/resources/custom_call_helper.py

```diff
@@ -1,6 +1,10 @@
+import os
 import ujson
+import geodata
 
-country_geojson_file = open("geodata/data_backup/country_data.json")
+geodata_path = os.path.abspath(geodata.__path__[0])
+country_geojson_file = open(
+    "{geodata}/data_backup/country_data.json".format(geodata=geodata_path))
 country_geojson = ujson.load(country_geojson_file)
```

Comment:
@Murrx  why not create helper function in `geodata.updaters` and move this code in there?
Additionally cleaning these lines:
https://github.com/openaid-IATI/OIPA/blob/develop/OIPA/geodata/updaters.py#L107-L108

Than you don't have to use `geodata.__path__` magic and data loading will be isolated in one place.